### PR TITLE
Feature/activity shortcodes

### PR DESCRIPTION
**Pull request by domjngo**

### CODE REVIEW

File: inc/functions-shortcode.php [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+			return embed_code( 'four-three', 'p02', 'beginner-activities', 'practice02.swf' );
+			break;
+		case '2':
+			return embed_code( 'four-three', '02', 'beginner-activities', 'activity02.swf' );
+			break;
+		case 'p3':
+			return embed_code( 'four-three', 'p03', 'beginner-activities', 'practice03.swf' );
+			break;
+		case '3':
+			return iframe_activity( '125%', '/beginner-activities/activity03.html' );
```
Comment (sdwilkes) [inline inc/functions-shortcode.php:27]: Flash files using iframes are not rendering at full width in Firefox and Chrome. They are full width in Safari however.

File: inc/functions-shortcode.php [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+						<PARAM NAME="SCALE" VALUE="exactfit">
+						<embed src="%s/%s" SCALE="exactfit" menu="false" wmode="transparent" name="activity-%s">
+					</object>
+				</div>
+			</div>';
+
+	return sprintf( $html, $class, $id, $dir, $file, $dir, $file, $id );
+}
+
+function iframe_activity( $ratio, $activity ) {
```
Comment (sdwilkes) [inline inc/functions-shortcode.php:109]: See comment about iframes not rendering at 100% in Firefox and Chrome.

**Review by sdwilkes — CHANGES_REQUESTED** [review]
Hi @domingobishop , I've added a couple of comments about iframes not rendering at 100% in Chrome and Firefox.

File: inc/functions-shortcode.php (hunk 1 above)
Comment (domjngo) [reply inc/functions-shortcode.php:27]: I think it's now fixed. CSS wasn't targeting embed code inside the iframe, so I added the CSS directly into the HTML doc.

File: inc/functions-shortcode.php (hunk 2 above)
Comment (domjngo) [reply inc/functions-shortcode.php:109]: See above

**Review by sdwilkes — APPROVED** [review]
Looks good Chris. Flash files are rendering at the correct width in Firefox and Chrome now.

**Pull request merged**